src/jquery.multiselect.js

```diff
@@ -469,25 +591,47 @@
       var self = this;
       var $button = this.$button;
       function clickHandler() {
-        self[ self._isOpen ? 'close' : 'open' ]();
-        return false;
+         self[ self._isOpen ? 'close' : 'open' ]();
+         return false;
       }
 
       $button
         .on({
           click: clickHandler,
-          keypress: function(e) {
-            switch(e.which) {
-              case 27: // esc
-              case 38: // up
-              case 37: // left
-                self.close();
-                break;
-              case 39: // right
-              case 40: // down
-                self.open();
-                break;
-            }
+          keydown: function(e) {
```

Comment:
Why the switch to keydown?

Reply:
@mlh758 Please read this:
[https://stackoverflow.com/questions/10191621/jquery-keypress-keydown-which](https://stackoverflow.com/questions/10191621/jquery-keypress-keydown-which)

Long story short, keypress was not working, so I had to switch to keydown.  
If you look at the button keyboard events, I believe it is keydown there also.

BTW, this caught me off-guard.  I had not encountered this difference before.

Reply:
Well alright. That is interesting, thanks for changing it then.